src/main/resources/lang/polish.lang

```diff
@@ -0,0 +1,190 @@
+			# 	stopka: Aby wyświetlić changelog wersji, wpisz <gold>/skript update changes <version><reset>
+			# nieprawidłowa wersja: Brak changelogu dla wersji <gold>%s<red> available
+			title: <bold><cyan>%s<reset> (%s)
+			next page: <grey>strona %s z %s. Wpisz <gold>/skript update changes %s<gray> na następną stronę (wskazówka: użyj klawisza strzałki w górę)
+	info:
+		aliases: Aliasy Skriptu można znaleźć tutaj.: <aqua>https://github.com/SkriptLang/skript-aliases
+		documentation: Dokumentację Skript można znaleźć tutaj: <aqua>https://docs.skriptlang.org/
+		tutorials: Tutoriale Skript można znaleźć tutaj: <aqua>https://docs.skriptlang.org/tutorials
+		version: Wersja Skript: <aqua>%s
+		server: Wersja serwera: <aqua>%s
+		addons: Installed Skript Addons: <aqua>%s
+		dependencies: Zainstalowane zależności: <aqua>%s
+
+# -- Updater --
+updater:
+	not started: Skript nie sprawdził jeszcze najnowszej wersji. Użyj <gold>/skript update check<reset> aby sprawdzić.
+	checking: Sprawdzanie czy jest najnowsza wersja Skript...
+	check in progress: Sprawdzanie nowej wersji jest obecnie w toku.
+	updater disabled: Aktualizator jest wyłączony, więc sprawdzenie najnowszej wersji Skript nie zostało wykonane.
+	check error: <red>Wystąpił błąd sprawdzania najnowszej wersji Skript:<light red> %s
+	running latest version: Obecnie używasz najnowszej stabilnej wersji Skript.
+	running latest version (beta): aktualnie masz włączoną <i>bete<r> wersji Skript, i żadnych nowych <i>stabilnych<r> wersja jest dostępna. Należy pamiętać, że aktualizację do nowszych wersji beta należy przeprowadzić ręcznie!
+	update available: Nowa wersja Skript jest dostępna: <gold>%s<reset> (obecnie prowadzisz <gold>%s<reset>)
+	downloading: Pobieranie Skryptu <gold>%s<reset>...
+	download in progress: Obecnie trwa pobieranie najnowszej wersji programu Skript.
+	download error: <red>Wystąpił błąd przy pobieraniu najnowszej wersji Skript.:<light red> %s
+	downloaded: Najnowsza wersja programu Skript została pobrana! Uruchom ponownie serwer lub użyj /reload aby zastosować zmiany.
+	internal error: Podczas sprawdzania najnowszej wersji Skript wystąpił wewnętrzny błąd. Szczegółowe informacje znajdują się w logu serwera.
+	custom version: Obecnie używasz niestandardowej wersji Skript. Żadne aktualizacje nie zostaną automatycznie zainstalowane.
+	nightly build: Obecnie używasz wersji rozwojowej programu Skript. Żadne aktualizacje nie zostaną automatycznie zainstalowane.
+
+# -- Commands --
+commands:
+	no permission message: Nie masz wymaganych uprawnień, aby użyć tego polecenia
+	cooldown message: Używasz tego polecenia zbyt często, spróbuj ponownie później
+	executable by players: Ta komenda może być użyta tylko przez graczy
+	executable by console: To polecenie może być użyte tylko przez konsolę
+	correct usage: Prawidłowe użycie:
+	invalid argument: Nieważny argument <gray>'%s<gray>'<reset>. Dozwolone są:
+	too many arguments: To polecenie może zaakceptować tylko jedno %2$s.
```

Comment:
Czemu akurat tylko jedno? To chyba wyświetla się też gdy np. przy 2 argumentach podasz 3 argumenty z tego co wiem

Reply:
Sprawdź dla angielskiego tłumaczenia

`too many arguments: This command can only accept a single %2$s.`

Jest to odnośnie wielu, ale pojedynczych argumentów